src/components/token-transfer.tsx

```diff
@@ -137,6 +138,20 @@ const ItemIcon = ({ type }: { type: EventSingle['event_type'] }) => {
       );
   }
 };
+
+const ItemThing = ({ data }: { data: any }) => {
+  if (data.contract_log) {
+    let value = data.contract_log.value;
+    const thing = deserializeCV(Buffer.from(data.contract_log.value.replace('0x', ''), 'hex'));
+
+    if (thing.type === 5 && 'address' in thing) {
```

Comment:
can you make a helper fn that describes what this does?

Reply:
Yeah def should have renamed this lol